prover/src/logup_gkr/mod.rs

```diff
@@ -56,6 +57,7 @@ pub struct EvaluatedCircuit<E: FieldElement> {
 impl<E: FieldElement> EvaluatedCircuit<E> {
     /// Creates a new [`EvaluatedCircuit`] by evaluating the circuit where the input layer is
     /// defined from the main trace columns.
+    #[instrument(skip_all)]
     pub fn new(
```

Comment:
Here, it might be good to add a custom name (similar to how we do it [here](https://github.com/facebook/winterfell/blob/c388e16acc866af6ca9477273b9c24dd201fc5ff/prover/src/constraints/evaluator/default.rs#L53)).

Reply:
Added!